src/components/Reports/ReportEdit.tsx

```diff
@@ -24,6 +26,8 @@ export default function ReportEdit(): JSX.Element {
   const snackbar = useSnackbar();
 
   const [report, setReport] = useState<Report>();
+
+  const [checkUserInterval, setCheckUserInterval] = useState<NodeJS.Timer>();
```

Comment:
we could use `ReturnType<typeof setInterval>` .. Charlie flagged a similar issue in my PR.. (NodeJS is more BE specific)..